typed-racket-test/unit-tests/shallow-rewrite-expansion/main.rkt

```diff
@@ -4,15 +4,14 @@
 ;; - fix datum-literals ?
 ;; - enable type-annotation test by upgrading user's annotations to trust codomain
 
-(require
-  "../test-utils.rkt"
-  rackunit
-  racket/pretty
-  racket/list
-  racket/set
-  syntax/parse
-  (only-in racket/format ~a)
-  (only-in syntax/modread with-module-reading-parameterization))
+(require racket/list
+         racket/pretty
+         racket/set
+         rackunit
+         syntax/parse
+         (only-in racket/format ~a)
+         (only-in syntax/modread with-module-reading-parameterization)
+         "../test-utils.rkt")
```

Comment:
This require form seemed to trip up Resyntax for some reason. The output looks nice, but Resyntax ran 10 passes and kept repeatedly trying to tidy these requires. Not sure what's up with that.